tests/assignOfflineContact.test.ts

```diff
@@ -1,4 +1,5 @@
 import { ServerlessCallback } from '@twilio-labs/serverless-runtime-types/types';
```

Comment:
So this test suite was actually failing locally for me when I ran the full test suite, but not when I ran the suite by itself. 1 of the 500 test conditions was failing (the one where 'task create' throws an error).

On further investigation, the test logic was actually broken, it's assertion was never passing (you can see the failed assertion in the logs for previous PR runs), but this was being covered up by the fact that the assertion was in a callback, and this was being caught by the callback handling code (still not sure why my local was sometimes failing though, seemed to be timing out?)

The root of the problem is the practice of putting assertions in callbacks. We should instead try to only capture the data to be asserted in callbacks, then assert on them after the test actions are complete. This makes the test easier to follow, as it keeps the normal 'Arrange, Act, Assert' code flow, rather than the assertions being before the test action, but more importantly it provents failed assertions being caught and swallowed by exception handling code in the part of the application that runs the callback.